src/NUnitFramework/tests/Attributes/TimeoutTests.cs

```diff
@@ -167,6 +181,49 @@ public void TimeoutWithMessagePumpShouldAbort()
             Assert.That(result.ResultState, Is.EqualTo(ResultState.Failure));
             Assert.That(result.Message, Is.EqualTo("Test exceeded Timeout value of 500ms"));
         }
+#endif
+
+#if !THREAD_ABORT
+        [Timeout(50)]
+        public void TestTimeoutAndReportsTimeoutFailure()
+        {
+            while (true) { }
```

Comment:
This is a busy loop that will eat CPU for the entire rest of the run of our test suite since we can't abort it. Would you change this to `Thread.Sleep(Timeout.Infinite);` please?

Reply:
Done